baselines/fedexp/fedexp/conf/base.yaml

```diff
@@ -0,0 +1,43 @@
+---
+seed: 0
+
+num_clients: 100
+num_rounds: 500
+clients_per_round: 20
+num_epochs: 20
+batch_size: 50
+
+server_device: cuda
+client_device: cuda
+
+client_resources:
+  num_cpus: 1
+  num_gpus: 0.05
```

Comment:
It might be wise raising this a bit. I'm not sure if everyone out there can fit 20 (1/20=0.05) clients on the same machine. Testing on my machine i see each client requires ~1.5GB of VRAM. I'm thinking better raise this to `0.1` and leave a comment in the README's "Hardware Setup" paragraph giving some advice on hot to adjust it.

Reply:
But very glad to see a baseline packing many clients on the same GPU 💯 :
<img width="643" alt="Screenshot 2023-09-25 at 22 14 36" src="https://github.com/adap/flower/assets/12958157/2046e293-a86b-43e7-886f-4000cdfee965">